File: src/main/webapp/js/service/informationService.js
//服务层
app.service('informationService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../information/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../information/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../information/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../information/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../information/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../information/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../information/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/top/axbt/keshe/service/InformationService.java
package top.axbt.keshe.service;

import top.axbt.entity.PageResult;
import top.axbt.keshe.domain.TbInformation;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface InformationService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbInformation> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbInformation information);
	
	
	/**
	 * 修改
	 */
	public void update(TbInformation information);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbInformation findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbInformation information, int pageNum, int pageSize);
	
}
